deploy/validateConfig.sh

```diff
@@ -0,0 +1,172 @@
+#!/bin/bash
```

Comment:
can you have variables store all args, and then use the variables? It'll improve readability and maintainability

Reply:
please remove all `` and replace it with $()

Reply:
@rjshrjndrn  Sorry I didn't get this point, you mean to create variables like implementation_name, database_host  etc and then get these values from the config file?
Yes ok `` will be replaced by $().

Reply:
So what I have planned is to have all mandatory values in an associative array like below -
arr[env]]=$(awk '/env: /{ if ($2 !~ /#.*/) {print $2}}' config)
Then iterate over this array and based on key value invoke the validation function.
If I use just plain variables like below, -
env_value=$(awk '/env: /{ if ($2 !~ /#.*/) {print $2}}' config) - This would require me to have too many if else blocks to check each value and when trying the display the message, we have to manually echo the key each time like echo "Invalid env: value", echo "Invalid database_host: value".
An array would be easier. What do you think?